src/config/configMaps.js

```diff
@@ -45,9 +45,9 @@ export const search = {
   sortSearchByDistance: false,
 
   // Limit location autocomplete to a one or more countries
-  // using ISO 3166 alpha 2 country codes separated by commas.
+  // using CLDR two-character region codes, separated by commas.
```

Comment:
This change sounds wrong.
- **_ISO 3166 alpha 2_** country code is a standard. 
  (CLDR two-character region code is likely following that standard too)
- Mapbox supports the standard, so we should not change the config away from it:
  https://docs.mapbox.com/api/search/geocoding/#forward-geocoding-with-search-text-input